app/controllers/entries_controller.rb

```diff
@@ -139,10 +139,12 @@ def confirm_to_white
 		dictionary = Dictionary.editable(current_user).find_by_name(params[:dictionary_id])
 		raise ArgumentError, "Cannot find the dictionary" if dictionary.nil?
 
-		entry = Entry.find(params[:id])
-		raise ArgumentError, "Cannot find the entry" if entry.nil?
+		raise ArgumentError, "No entry to be confirmed is selected" unless params[:entry_id].present?
 
-		dictionary.confirm_entry(entry)
+		ActiveRecord::Base.transaction do
+			entries = Entry.where(id: params[:entry_id])
+			entries.each{|entry| dictionary.confirm_entry(entry)}
```

Comment:
```suggestion
			entries = Entry.where(id: params[:entry_id])
			entries.each{|entry| entry.be_white!}
			update_entries_num
```

毎回update_entries_numしなくていいので、こうした方が良さそうです。

ただし、ロジックはコントローラーよりモデルに書く方がいいです。
この処理を行う`Dictionary#confirm_entries`があると良さそうです。

Reply:
ありがとうございます。
[確定ロジックをconfirm_entriesメソッドに変更](https://github.com/pubannotation/pubdictionaries/pull/112/commits/e594fe4308f98a273e0fd5e5c6a89e73cd375309)にて、`Dictionary#confirm_entries`を作ってロジックをモデルに移動させ、元の`confirm_entry`は不要になったので削除しました。